### PR TITLE
Report test results to the discord embeds

### PR DESCRIPTION
**Pull request by skhomuti**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@ name: Test wallets
 on:
   pull_request:
   schedule:
-    - cron:  '0 2,14 * * *'
+     - cron: '0 */6 * * *'
+
 jobs:
   test-packages:
```
Comment (dgusakov) [inline .github/workflows/test.yml:8]: Need more representative name here

File: wallets-testing/discord-reporter.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  TestCase,
+  TestResult,
+} from '@playwright/test/reporter';
+import * as fs from 'fs';
+
+const testStatusToEmoji = {
+  passed: '✅',
+  failed: '❌',
+  timedOut: '⏰',
+  skipped: '⏭️',
```
Comment (dgusakov) [inline wallets-testing/discord-reporter.ts:13]:
```suggestion
  skipped: '⏸️',
```

File: wallets-testing/discord-reporter.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  interrupted: '🚫',
+};
+
+const successField = {
+  name: '**💆‍♂️️ Periodically running wallet tests pass**',
```
Comment (dgusakov) [inline wallets-testing/discord-reporter.ts:18]:
```suggestion
  name: '**💆‍♂️️ Periodically running wallet tests PASSED**',
```

File: wallets-testing/discord-reporter.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  value: `[🔗 Tests summary](${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}/actions/runs/${process.env.GITHUB_RUN_ID})`,
+  inline: false,
+};
+
+const failedField = {
+  name: '🙅‍♂️ Periodically running wallet tests failed',
```
Comment (dgusakov) [inline wallets-testing/discord-reporter.ts:24]:
```suggestion
  name: '🙅‍♂️ Periodically running wallet tests FAILED',
```

File: wallets-testing/discord-reporter.ts (hunk 2 above)
Comment (skhomuti) [reply wallets-testing/discord-reporter.ts:13]: how dare you argue with copilot

**Pull request merged**